content/docs/docs-nav.yaml

```diff
@@ -7,6 +7,7 @@ plugins:
 nav:
   - Getting started:
     - Installing the GitHub app: '/docs/getting-started/install-github-app/'
+    - Configure GitHub Enterprise: '/docs/getting-started/github-enterprise/'
```

Comment:
95% of users won't have to do this step so the Getting Started guide is not the right place.

Could you put it in Details please and then link to it on the "Installing the GitHub app" page? Something like... "If you use GitHub Enterprise hosted in a cloud environment like AWS or Azure..."

Reply:
done.